compiler/luci-interpreter/src/kernels/TransposeConv.h

```diff
@@ -34,6 +34,7 @@ class TransposeConv : public KernelWithParams<TransposeConvParams>
   const Tensor *output_shape() const { return _inputs[0]; }
   const Tensor *filter() const { return _inputs[1]; }
   const Tensor *input() const { return _inputs[2]; }
+  const Tensor *bias() const { return nullptr; }
```

Comment:
Please add another parameter to the constructor (bias) and return it here.
You will also need to update tests (including KernelBuilder.test.cpp).

Reply:
Okay, but currently luci TransposeConv node only have arity 3. so on `KernelBuilder.cpp`, I will pass a bias nullptr. after update, luci, I will update `KernelBuilder.cpp` to pass 4th Tensor, `KernelBuilder.test.cpp` to pass bias from Circle node and finally `TransposeConv.ccp` to handle bias.

Reply:
I will Add all context above and split into sub PR and update it. Thanks!